frontend/awx/views/schedules/SchedulePage/ScheduleDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTranslation } from 'react-i18next';
+import { PageDetail, PageDetails } from '../../../../../framework';
+import { UserDateDetail } from '../../../common/UserDateDetail';
+import { Schedule } from '../../../interfaces/Schedule';
+import { formatDateString } from '../../../../../framework/utils/formatDateString';
+
```

Comment:
This component is intentionally incomplete and will be built out as the rest of the schedule workflow is built out.